src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -616,6 +622,8 @@ public static SyntaxKind GetBinaryExpression(SyntaxKind token)
                     return SyntaxKind.LogicalAndExpression;
                 case SyntaxKind.BarBarToken:
                     return SyntaxKind.LogicalOrExpression;
+                case SyntaxKind.DotDotToken:
```

Comment:
>case SyntaxKind.DotDotToken: [](start = 16, length = 28)

It feels unexpected to handle SyntaxKind.DotDotToken in this function, RangeExpression is not a BinaryExpressionSyntax.